### ISSUE TITLE
ResharperParser throws exception if message is empty

### ISSUE THREAD

**Issue by uhafner** [tomasbjerre/violations-lib#78]
ResharperParser throws exception if message is empty:
```
RuntimeException: "Message" not found in "<Issue TypeId="CppCompilerErrors" File="..\trunk\xxx\yyy\zzz.h" Offset="1198-1199" Line="74" Message="" />"
java.lang.RuntimeException: "Message" not found in "<Issue TypeId="CppCompilerErrors" File="..\trunk\xxx\yyy\zzz.h" Offset="1198-1199" Line="74" Message="" />"
	at se.bjurr.violations.lib.util.ViolationParserUtils.getAttribute(ViolationParserUtils.java:72)
	at se.bjurr.violations.lib.parsers.ResharperParser.parseReportOutput(ResharperParser.java:45)
	at edu.hm.hafner.analysis.parser.violations.AbstractViolationAdapter.parse(AbstractViolationAdapter.java:33)
	at io.jenkins.plugins.analysis.core.model.FilesScanner.aggregateIssuesOfFile(FilesScanner.java:105)
	at io.jenkins.plugins.analysis.core.model.FilesScanner.scanFiles(FilesScanner.java:89)
	at io.jenkins.plugins.analysis.core.model.FilesScanner.invoke(FilesScanner.java:72)
	at io.jenkins.plugins.analysis.core.model.FilesScanner.invoke(FilesScanner.java:30)
	at hudson.FilePath$FileCallableWrapper.call(FilePath.java:3041)
	at hudson.remoting.UserRequest.perform(UserRequest.java:212)
	at hudson.remoting.UserRequest.perform(UserRequest.java:54)
	at hudson.remoting.Request$2.run(Request.java:369)
	at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:72)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at hudson.remoting.Engine$1.lambda$newThread$0(Engine.java:93)
	at java.lang.Thread.run(Thread.java:748)
```

For details see [JENKINS-59715](https://issues.jenkins-ci.org/browse/JENKINS-59715)

**Comment by tomasbjerre** [comment]
Released a fix in 1.104 released to JCenter. Will also release to Central, but Sonatype is having problems now so might take a while.

**Comment by uhafner** [comment]
Thanks for the fast fix!
